cmd/system_test.go

```diff
@@ -35,8 +35,12 @@ func (m *outputMonitor) Write(p []byte) (int, error) {
 }
 
 func startDaemon(port int) (*daemon, error) {
-	addr := fmt.Sprintf("127.0.0.1:%d", port)
-	cmd := exec.Command("levityd", addr)
+	addr := fmt.Sprintf("0.0.0.0:%d", port)
```

Comment:
Why bind `0.0.0.0`?  Seems dangerous.

Reply:
Interesting approach to integration testing, I would have just created and started the server process, but this tests process startup like the user would start it which is nice.

Agree with @fspmarshall though, tests should run on loopback. Also might want to do `127.0.0.1:0` to let the OS select a high random port.

Reply:
Done